operator_ui/src/screens/Bridge/BridgeScreen.test.tsx

```diff
@@ -146,7 +126,6 @@ describe('BridgeScreen', () => {
           },
         },
       },
-      refetchBridgesQuery(),
```

Comment:
No longer need to refresh since every load of the Bridges page will now make a new network request